Makefile

```diff
@@ -241,6 +243,82 @@ catalog-deploy:
 	sed -e 's~<IMG>~$(CATALOG_IMG)~' ./config/samples/catalog/catalog.yaml | kubectl apply -f -
 
 # Deploy the sample FlowCollector CR
-.PHONY: create-sample
-create-sample:
-	sed -e 's~:main~:v$(VERSION)~' ./config/samples/flows_v1alpha1_flowcollector.yaml | kubectl apply -f -
+.PHONY: deploy-sample-cr
+deploy-sample-cr:
+	sed -e 's~:main~:$(VERSION)~' ./config/samples/flows_v1alpha1_flowcollector.yaml | kubectl apply -f - || true
+
+# Undeploy the sample FlowCollector CR
+.PHONY: undeploy-sample-cr
+undeploy-sample-cr:
+	sed -e 's~:main~:$(VERSION)~' ./config/samples/flows_v1alpha1_flowcollector.yaml | kubectl --ignore-not-found=true delete -f -  || true
+
+##@ Development
+
+.PHONY: deploy-loki
+deploy-loki: ## Deploy loki.
+	@echo "### Deploying loki"
+	kubectl create namespace $(NAMESPACE)  --dry-run=client -o yaml | kubectl apply -f -
```

Comment:
You can just do this.

```
kubectl create namespace $(NAMESPACE)
```

Reply:
@stleerh Yes ... but this will fail if the namespace exists... so this is like the recommended way to make sure that we create the namespace only if it doesn't exist. Another option is to add the regular || true ... but this is in "the eyes of the beholder"  on how to implement this  :-)